components/switch/overview.md

```diff
@@ -12,10 +12,10 @@ position: 0
 
 The <a href = "https://www.telerik.com/blazor-ui/switch" target="_blank">Blazor Switch component</a> allows the user to toggle between checked and unchecked states.
 
-#### To use a Telerik Switch for Blazor
+## Creating Switch for Blazor
 
-1. add the `TelerikSwitch` tag
-1. provide `Value` (one-way data binding) or `bind-Value` (two-way data binding) property
+1. Use the `TelerikSwitch` tag
+1. Provide `Value` (one-way data binding) or `bind-Value` (two-way data binding) property
```

Comment:
Or you can just say "Set the `Value` parameter to a `bool` property. It supports one-way or two-way binding."
```suggestion
1. Provide `Value` (one-way data binding) or `@bind-Value` (two-way binding) property
```